kvbc/include/db_interfaces.h

```diff
@@ -34,7 +35,9 @@ class ILocalKeyValueStorageReadOnly {
  */
 class IBlocksAppender {
  public:
-  virtual concordUtils::Status addBlock(const SetOfKeyValuePairs& updates, BlockId& outBlockId) = 0;
+  virtual concordUtils::Status addBlock(const SetOfKeyValuePairs& updates,
+                                        BlockId& outBlockId,
+                                        const concordUtils::SpanWrapper& parent_span = concordUtils::SpanWrapper{}) = 0;
```

Comment:
Can we possibly consider using overloads (one with and one without the span) instead of default parameters for virtual methods? It might make the code more generic, clearer and safer.